eoscostarica/eosio-dashboard
# reduce final bundle size

### reduce final bundle size 

### What does this PR do?
Avoid load unused icons from ual-reactjs-renderer to reduce the final bundle size
for more information take a look to https://github.com/EOSIO/ual-reactjs-renderer/issues/103 and https://github.com/EOSIO/ual-reactjs-renderer/pull/105

### Steps to test

1. make run lacchain
2. navigate to http://localhost:3000/
3. check the loading time

#### CheckList
